### ISSUE TITLE
Resume after connection got interrupted

### ISSUE THREAD

**Issue by orschiro** [Quacky2200/Spotify-Web-Player-for-Linux#83]
Dear all,

I need to restart the application in order to resume the playback after an internet connection interruption. 

The application does not automatically resume the playback after reconnect. 

Can someone confirm this problem?

Yours,

Robert

**Comment by Quacky2200** [comment]
Yes this may happen. I did put in a piece of code that would restart the web page but Spotify doesn't always detect if no internet connection is present and sometimes a restart is required. This is more of a bug with Spotify than mine as they should be able to make sure that it reconnects when the internet connection is patchy but they haven't. A lot of their code is also very condensed and complicated to use, a work around would be hard to be put in place because the issue is with their web sockets.

**Comment by orschiro** [comment]
I see. 

Thanks for the background information. 

Let's keep it as is then, I guess...
